File: Chapter 03/Python Code/3.13.py
n = int(input("Enter the number of terms: "))

sum = 0
f=1

for i in range(1, n+1):
    if i == 1:
        sum += 1/2
    else:
        f = f*(i+1)
        sum += i/f

print("Sum of the series is:", sum)

File: Chapter 03/C Code/3_13.c
//3.13
#include <stdio.h>

int main() {
int n;
double sum = 0, factorial = 1;
printf("Enter the value of n: ");
scanf("%d", &n);

for(int i = 1; i <= n; i++) {
factorial *= i+1;
sum += i / factorial;
}

printf("Sum of the series is: %lf", sum);
return 0;
}
